requirements/pip.txt

```diff
@@ -10,5 +10,7 @@ wheel==0.37.1
 # The following packages are considered to be unsafe in a requirements file:
 pip==22.1.2
     # via -r requirements/pip.in
-setuptools==62.4.0
-    # via -r requirements/pip.in
+setuptools==59.8.0
```

Comment:
Downgrading the version because of the introduction of common constraints in the `pip.in` file.